arrows/qt/widgets/LoggerView.cxx

```diff
@@ -0,0 +1,69 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include "LoggerView.h"
+#include "ui_LoggerView.h"
+
+#include <qtStlUtil.h>
+
+namespace kv = kwiver::vital;
+
+namespace kwiver {
+namespace arrows {
+namespace qt {
+
+//-----------------------------------------------------------------------------
+class LoggerViewPrivate
+{
+public:
+  Ui::LoggerView m_UI;
+};
+
+QTE_IMPLEMENT_D_FUNC(LoggerView)
+
+//-----------------------------------------------------------------------------
+LoggerView::LoggerView(
+  QWidget* parent, Qt::WindowFlags flags)
+  : QWidget{parent, flags}, d_ptr{new LoggerViewPrivate}
+{
+  QTE_D();
+
+  // Set up UI
+  d->m_UI.setupUi(this);
+
+  QFont font{ "Monospace", 10 };
```

Comment:
At this point it's probably not worth worrying about, but there *are* extra spaces here (which, technically, are recommended by the KWIVER style guide) that aren't present elsewhere :slightly_smiling_face:.